<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMajorData</class>
 <widget class="QDialog" name="DialogMajorData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*pushbutton */
QPushButton {
border: 0.5px solid white;
border-radius: 6px;
background-color: rgb(90,194,198);
min-width: 80px;
font-family: &quot;Microsoft YaHei&quot;;
font-size:11pt;
font-weight: bold;
color:white;
}

QPushButton:hover {
border: 0.5px solid white;
border-radius: 6px;
background-color: #1fab89;
min-width: 80px;
font-family: &quot;Microsoft YaHei&quot;;
font-size:10pt;
font-weight: bold;
color:white;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="1">
       <widget class="QComboBox" name="comboCollegeId"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>学院编号：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboCollegeName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>专业名称：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leMajorName"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>专业编号：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>学院名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leMajorId"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnCommit">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogMajorData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCommit</sender>
   <signal>clicked()</signal>
   <receiver>DialogMajorData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
